handbook/company/communications.md

```diff
@@ -626,9 +626,28 @@ At the end of their first two weeks of onboarding at Fleet, every new team membe
 
 Fleet prioritizes a [bias for action](https://fleetdm.com/handbook/company#ownership).  If possible, apply onboarding feedback to the handbook and issue templates in realtime, during this call.  This avoids backlogging tasks that may just get out of date before we get around to them anyway.
 
-## Receiving performance feedback
+## Performance feedback
```

Comment:
Changing this to "Performance feedback" because the section is now about both giving and receiving feedback.